backend/api/Models/Concepts/Document/UpdateMetadata/DocumentUpdateMetadataRequest.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.AspNetCore.Http;
+using System.Collections.Generic;
+
+namespace Pims.Api.Models.Concepts
+{
+    /// <summary>
+    /// DocumentUpdateMetadataRequest class, represents a request to update a document metadata.
+    /// </summary>
+    public class DocumentUpdateMetadataRequest
```

Comment:
How different is the create from the update? could we use one modal to represent both?

Reply:
Create request contains type ids but update request contains primary key ids, it will be confusing if we use the same model.